sdcm/tester.py

```diff
@@ -1151,6 +1151,7 @@ def calculate_stats_average(self):
             self.stats['results']['stats_total'] = total_stats
 
     def update_test_details(self, snapshot_uploaded=False):
+        return
```

Comment:
Which fix is this related?

Reply:
Sorry, I pushed a debug patch in this PR, will remove it.